### ISSUE TITLE
How to stop environment initialization?

### ISSUE THREAD

**Issue by hbaniecki** [rstudio/reticulate#894]
I was using `reticulate` with normal Python and without any special environments; then installed `miniconda`. Now, the first time when I use `py_load_object()` I see some `r-reticulate` environment initialization. How to stop this automatic behaviour? 

https://github.com/rstudio/reticulate/blob/d14c0f00ba3a665281bafc2e672a341adf8ef235/R/config.R#L204-L216

I believe this makes the `py_load_object()` use the newly created `r-reticulate` environment instead of native Python.

Basically, it would be useful to be able to set some variable that would set `conda_installed()` to `FALSE`.

**Comment by kevinushey** [comment]
For `py_load_object()` to work, Python needs to be initialized. You can control what edition of Python is initialized via `use_python()`, `use_virtualenv()`, and so on. Those need to be called before invoking any function that would force initialization of Python.

**Comment by hbaniecki** [comment]
I had Python installed and didn't use any of the `use_python()` or `use_virtualenv()` commands. After the `miniconda` installation (not for the `reticulate` usage) I need to apply `use_python()` to override the default behaviour (which is: make `r-reticulate` env). 

Anyway, this can be closed if this behaviour only bothers me.
